uaclient/cli/vulnerability/update.py

```diff
@@ -1,7 +1,5 @@
 from uaclient import messages
```

Comment:
why was `update` left behind if we're deleting the other two vulnerability commands in this PR?